mypy-python/action.yaml

```
@@ -0,0 +1,26 @@
+name: "Python Mypy Action"
+author: "Tom Ricciuti <tom.ricciuti@greenbone.net>"
+description: "An action that verifies python type hints by using mypy"
+inputs:
+  packages:
+    description: "Python packages to check with mypy"
+    required: true
+  version:
+    description: "Python version that should be installed"
+    default: 3.9
+branding:
+  icon: "package"
+  color: "green"
+runs:
+  using: "composite"
+  steps:
+    - name: Install poetry
+      uses: greenbone/actions/poetry@v1
```

Must be main in main branch and v2 in v2 branch.
```suggestion
      uses: greenbone/actions/poetry@main
```